sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/documentanalysis/models/DocumentSelectionMarkState.java

```diff
@@ -11,6 +11,13 @@
 /** Defines values for DocumentSelectionMarkState. */
 @Immutable
 public final class DocumentSelectionMarkState extends ExpandableStringEnum<DocumentSelectionMarkState> {
+
+    /**
+     * Creates a DocumentSelectionMarkState object.
+     */
+    public DocumentSelectionMarkState() {
```

Comment:
Since this is an `ExpandableStringEnum` this will need to be marked as deprecated and tell users to use the factory method.